ibis/backends/base/sql/alchemy/__init__.py

```diff
@@ -287,19 +302,16 @@ def _table_from_schema(
         if temp:
             prefixes.append('TEMPORARY')
         columns = self._columns_from_schema(name, schema)
-        return sa.Table(name, self.meta, *columns, prefixes=prefixes)
+        return sa.Table(name, sa.MetaData(), *columns, prefixes=prefixes)
```

Comment:
Why this change?

Reply:
I'm not sure this will be necessary after #5741. I'll confirm and report back here once that's merged.

Reply:
This is necessary to avoid this function looking at existing tables in metadata. This function is just for constructing an unbound `sa.Table` without knowledge of what's already in the database.

Reply:
I ended up removing the `meta` instance attribute. It was causing annoying problems with invalidating identifiers from snowflake. I added a bit of prose in the commit message about it.